src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -311,20 +311,22 @@ std::unique_ptr<model_backend> tf_compute_context::create_object_detector(
 }
 
 std::unique_ptr<model_backend> tf_compute_context::create_activity_classifier(
-    int n, int c_in, int h_in, int w_in, int c_out, int h_out, int w_out,
-    const float_array_map& config, const float_array_map& weights) {
-  shared_float_array prediction_window = config.at("ac_pred_window");
-  const float* pred_window = prediction_window.data();
-  int pw = static_cast<int>(*pred_window);
+    ac_parameters ac_params) {
+  // shared_float_array prediction_window = config.at("ac_pred_window");
```

Comment:
Should we just delete this code rather than comment it out?

Reply:
Yeah I should do that. I also should add comments I think. I will change it.